micro_benchmarking_pytorch.py

```diff
@@ -186,14 +191,16 @@ def run_benchmarking(local_rank, ngpus, net, batch_size, iterations, run_fp16, d
       print ("Num devices: 1")
     else:
       print ("Num devices: {}".format(ngpus))
-    print ("Mini batch size [img] : {}".format(batch_size))
-    print ("Time per mini-batch : {}".format(time_per_batch))
-    print ("Throughput [img/sec] : {}".format(batch_size/time_per_batch))
+      print ("Dtype: {}".format(dtype))
+      print ("Mini batch size [img] : {}".format(batch_size))
+      print ("Time per mini-batch : {}".format(time_per_batch))
+      print ("Throughput [img/sec] : {}".format(batch_size/time_per_batch))
```

Comment:
```suggestion
    print ("Dtype: {}".format(dtype))
    print ("Mini batch size [img] : {}".format(batch_size))
    print ("Time per mini-batch : {}".format(time_per_batch))
    print ("Throughput [img/sec] : {}".format(batch_size/time_per_batch))
```

Reply:
I don't think you were supposed to indent these statements into the `else` clause above.  Was this an accident?